### PR TITLE
feat: first iteration of the AACOT'23 website implementation

### PR DESCRIPTION
**Pull request by AceTheCreator**
This is the first implementation of the AsyncAPI conference website which includes the following.

- Index page
- Venues Page
- Speakers Component
- Venue Component
- The Globe with the pinned event location

cc @thulieblack @Mayaleeeee @derberg @Barbanio @alequetzalli

### CODE REVIEW

**Comment by thulieblack** [comment]
This looks 🤩🤩🤩🤩🤩🤩

File: config/city-lists.json [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+		"img": "https://images.unsplash.com/photo-1596176530529-78163a4f7af2?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2454&q=80",
+		"address": "309, Venkatesh complex, floor, 100 feet road, Indiranagar, Bangalore, Karnataka 560038, India"
+	},
+	{
+		"name": "Paris",
+		"country": "France",
+		"date": "TBH",
+		"description": "Join us in Madrid for AsyncAPI Conference and learn how to speak fluent API! Let's taco 'bout messaging and have a fiesta you won't forget!",
+		"img": "https://images.unsplash.com/photo-1508050919630-b135583b29ab?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2306&q=80",
+		"address": "TBH"
```
Comment (thulieblack) [inline config/city-lists.json:34]:
```suggestion
		"address": "TBA"
```

File: config/city-lists.json [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+	},
+	{
+		"name": "Melbourne",
+		"country": "Australia",
+		"date": "TBH",
+		"description": "Join us in Madrid for AsyncAPI Conference and learn how to speak fluent API! Let's taco 'bout messaging and have a fiesta you won't forget!",
+		"img": "https://images.unsplash.com/photo-1470294402047-fc1b5f39bd99?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2374&q=80",
+		"address": "TBH"
+	}
```
Comment (thulieblack) [inline config/city-lists.json:43]:
```suggestion
	
```

File: config/city-lists.json [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+		"address": "309, Venkatesh complex, floor, 100 feet road, Indiranagar, Bangalore, Karnataka 560038, India"
+	},
+	{
+		"name": "Paris",
+		"country": "France",
+		"date": "TBH",
+		"description": "Join us in Madrid for AsyncAPI Conference and learn how to speak fluent API! Let's taco 'bout messaging and have a fiesta you won't forget!",
+		"img": "https://images.unsplash.com/photo-1508050919630-b135583b29ab?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2306&q=80",
+		"address": "TBH"
+	},
```
Comment (thulieblack) [inline config/city-lists.json:35]:
```suggestion
	}
```

File: config/city-lists.json [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+	{
```
Comment (thulieblack) [inline config/city-lists.json:36]:
```suggestion
	
```

File: config/city-lists.json [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+		"name": "Melbourne",
```
Comment (thulieblack) [inline config/city-lists.json:37]:
```suggestion
```

File: config/city-lists.json [hunk 6]
```diff
@@ -0,0 +1,44 @@
...
+		"country": "Australia",
```
Comment (thulieblack) [inline config/city-lists.json:38]:
```suggestion
```

File: config/city-lists.json [hunk 7]
```diff
@@ -0,0 +1,44 @@
...
+		"date": "TBH",
+		"description": "Join us in Madrid for AsyncAPI Conference and learn how to speak fluent API! Let's taco 'bout messaging and have a fiesta you won't forget!",
```
Comment (thulieblack) [inline config/city-lists.json:40]:
```suggestion

```

File: config/city-lists.json [hunk 8]
```diff
@@ -0,0 +1,44 @@
...
+		"img": "https://images.unsplash.com/photo-1470294402047-fc1b5f39bd99?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2374&q=80",
```
Comment (thulieblack) [inline config/city-lists.json:41]:
```suggestion

```

File: config/city-lists.json [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+		"address": "TBH"
```
Comment (thulieblack) [inline config/city-lists.json:42]:
```suggestion

```

File: config/links.json [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+				"ref": "/venue/Bangalore"
+			},
+			{
+				"title": "Madrid, Spain",
+				"ref": "/venue/Madrid"
+			},
+			{
+				"title": "Paris, France",
+				"ref": "/venue/Paris"
+			},
```
Comment (thulieblack) [inline config/links.json:21]:
```suggestion
			}
```

File: config/links.json [hunk 11]
```diff
@@ -0,0 +1,36 @@
...
+			{
+				"title": "Melbourne, Australia",
```
Comment (thulieblack) [inline config/links.json:23]:
```suggestion
```

File: config/links.json [hunk 12]
```diff
@@ -0,0 +1,36 @@
...
+			},
+			{
+				"title": "Madrid, Spain",
+				"ref": "/venue/Madrid"
+			},
+			{
+				"title": "Paris, France",
+				"ref": "/venue/Paris"
+			},
+			{
```
Comment (thulieblack) [inline config/links.json:22]:
```suggestion
```

File: config/speakers.json [hunk 13]
```diff
@@ -0,0 +1,63 @@
...
+	{
+		"city": "Paris",
+		"location": "Paris, France",
+		"lists": []
+	},
+	{
+		"city": "Bangalore",
+		"location": "Bangalore, India",
+		"lists": []
+	},
```
Comment (thulieblack) [inline config/speakers.json:57]:
```suggestion
	}
```

File: config/speakers.json [hunk 14]
```diff
@@ -0,0 +1,63 @@
...
+	{
+		"city": "Melbourne",
+		"location": "Melbourne, Australia",
+		"lists": []
```
Comment (thulieblack) [inline config/speakers.json:61]:
```suggestion
```

File: config/speakers.json [hunk 15]
```diff
@@ -0,0 +1,63 @@
...
+	}
```
Comment (thulieblack) [inline config/speakers.json:62]:
```suggestion
```

**Review by thulieblack — COMMENTED** [review]
We'll be doing 4 events this year instead of 5 thus why we not going to include Melbourne

File: config/city-lists.json (hunk 1 above)
Comment (thulieblack) [inline config/city-lists.json:34]:
```suggestion
		"address": "Centre des Nouvelles Industries et Technologies, 2 Place de la Défense, 92092 Puteaux, Paris"
```

File: config/city-lists.json [hunk 16]
```diff
@@ -0,0 +1,44 @@
...
+		"date": "TBH",
+		"description": "Join us in Madrid for AsyncAPI Conference and learn how to speak fluent API! Let's taco 'bout messaging and have a fiesta you won't forget!",
+		"img": "https://images.unsplash.com/photo-1596176530529-78163a4f7af2?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2454&q=80",
+		"address": "309, Venkatesh complex, floor, 100 feet road, Indiranagar, Bangalore, Karnataka 560038, India"
+	},
+	{
+		"name": "Paris",
+		"country": "France",
+		"date": "TBH",
+		"description": "Join us in Madrid for AsyncAPI Conference and learn how to speak fluent API! Let's taco 'bout messaging and have a fiesta you won't forget!",
```
Comment (thulieblack) [inline config/city-lists.json:32]:
```suggestion
		"description": "Join us in Paris for the AsyncAPI Conference and learn more about APIs and EDAs! Let's have a memorable fiesta!",
```

File: config/city-lists.json [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+		"img": "https://images.unsplash.com/photo-1539037116277-4db20889f2d4?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1470&q=80",
+		"address": "Sngular Madrid, C. de Labastida, 1, 28034 Madrid, Spain",
+		"sponsors": ["/img/sngular.png"]
+	},
+	{
+		"name": "Bangalore",
+		"country": "India",
+		"date": "TBH",
+		"description": "Join us in Madrid for AsyncAPI Conference and learn how to speak fluent API! Let's taco 'bout messaging and have a fiesta you won't forget!",
```
Comment (thulieblack) [inline config/city-lists.json:24]:
```suggestion
		"description": "Join us in Bangalore for the AsyncAPI Conference and learn more about APIs and EDAs! Let's have a memorable fiesta!",
```

**Comment by AceTheCreator** [comment]
PR updated @thulieblack

File: config/city-lists.json [hunk 18]
```diff
@@ -0,0 +1,36 @@
...
+		"date": "September 26, 2023",
+		"description": "Join us in London for AsyncAPI Conference and learn how to speak fluent API! Let's taco 'bout messaging and have a fiesta you won't forget!",
+		"img": "/img/london.jpeg",
+		"address": "IBM UK Limited 20 York Road London SE1 7ND",
+		"sponsors": ["/img/IBM.png"]
+	},
+	{
+		"name": "Madrid",
+		"country": "Spain",
+		"date": "October 26, 2023",
```
Comment (thulieblack) [inline config/city-lists.json:14]:
```suggestion
		"date": "October 18-19, 2023",
```

File: config/city-lists.json [hunk 19]
```diff
@@ -0,0 +1,36 @@
+[
+	{
+		"name": "London",
+		"country": "UK",
+		"date": "September 26, 2023",
```
Comment (thulieblack) [inline config/city-lists.json:5]:
```suggestion
		"date": "September 20, 2023",
```

**Review by thulieblack — CHANGES_REQUESTED** [review]
1. Let's remove Melbourne from the venue page and add London
image
2. Lets remove this from the London Speakers page 
image
3. Correct the dates for London - 20th of September Madrid is either 18 or 19 October _(will update later on)_
4. This is a suggestion. Is there a way we can fill the upper part of the rotating globe? Instead of seeing half, it'll be full? 
image
5. When using a mobile phone the globe is not visible too

File: config/links.json [hunk 20]
```diff
@@ -0,0 +1,36 @@
...
+		"ref": "",
+		"subMenu": [
+			{
+				"title": "Bengaluru, India",
+				"ref": "/venue/Bangalore"
+			},
+			{
+				"title": "Madrid, Spain",
+				"ref": "/venue/Madrid"
+			},
```
Comment (thulieblack) [inline config/links.json:17]:
```suggestion
			},
			{
				"title": "London, UK",
				"ref": "/venue/London"
			},
```

**Comment by AceTheCreator** [comment]
> This is a suggestion. Is there a way we can fill the upper part of the rotating globe? Instead of seeing half, it'll be full?

This was intentional for aesthetics. 

> When using a mobile phone the globe is not visible too

This is also intentional as we need the site to load as fast as possible on mobile devices.

**Comment by Mayaleeeee** [comment]
Thanks for working on this @AceTheCreator, you did a fantastic job, and I love it. However, I have a few comments based on what I have noticed. 

- There is no interaction on the card here (e.g. hovered and selected)

Screenshot 2023-08-01 103208

- The texts under the location here are supposed to be centred-aligned, not left-aligned
Screenshot 2023-08-01 103430

Under this same page, we also mentioned that we should add a section where users will see the event/host sponsors of the location immediately after they enter the page, it's on the design I created (see the picture below), but it's not on the deploy-preview

Screenshot 2023-08-01 104141

- Also, the links here should take us to our social media pages
Screenshot 2023-08-01 104326

- The texts here are not well aligned, and even though they are still the same thing, the first one is 
Screenshot 2023-08-01 104749
under London, UK (looks bigger than the rest of it).

I think that's all I have noticed for now; if I see anything else, I will update this thread. 

Please let me know if there's anything you would like me to do from the design aspect. Thank you so much  @AceTheCreator

**Comment by AceTheCreator** [comment]
>under London, UK (looks bigger than the rest of it).

I don't understand what you're talking about

**Comment by thulieblack** [comment]
@AceTheCreator here is something for the Madrid page https://www.youtube.com/watch?v=SyZpHbWK5b4

**Comment by Mayaleeeee** [comment]
> > Under this same page, we also mentioned that we should add a section where users will see the event/host sponsors of the location immediately after they enter the page, it's on the design I created (see the picture below), but it's not on the deploy-preview
> 
> @Mayaleeeee It doesn't make sense to me why we are gonna have the sponsors there and then still have it in another section of the page. That was why I didn't include it.
> 
> The remaining you pointed out(apart from the last one) was all intentional as I also want other contributors to fix this minor issue as a means of contributing to the project

Oh, that's fine then,

**Comment by Mayaleeeee** [comment]
> > under London, UK (looks bigger than the rest of it).
> 
> I don't understand what you're talking about

Hey @AceTheCreator, I'm trying to explain that "Join us in London for AsyncAPI ..." appears larger than the sentences beneath Madrid and Bangaluru in the picture below. Despite containing the exact text, the London sentence seems larger. Also, you'll notice that the London sentence covers four lines, while the others only cover three. Do you now understand my point?

257476100-d285fecc-c666-475b-aecd-0d8639388e14

**Comment by AceTheCreator** [comment]
> > > under London, UK (looks bigger than the rest of it).
> > 
> > 
> > I don't understand what you're talking about
> 
> Hey @AceTheCreator, I'm trying to explain that **"Join us in London for AsyncAPI ..."** appears larger than the sentences beneath Madrid and Bangaluru in the picture below. Despite containing the exact text, the London sentence seems larger. Also, you'll notice that the London sentence covers four lines, while the others only cover three. Do you now understand my point?
> 
> 257476100-d285fecc-c666-475b-aecd-0d8639388e14

@Mayaleeeee the text beneath London is not the same length as the rest

**Comment by Mayaleeeee** [comment]
> > > > under London, UK (looks bigger than the rest of it).
> > > 
> > > 
> > > I don't understand what you're talking about
> > 
> > Hey @AceTheCreator, I'm trying to explain that **"Join us in London for AsyncAPI ..."** appears larger than the sentences beneath Madrid and Bangaluru in the picture below. Despite containing the exact text, the London sentence seems larger. Also, you'll notice that the London sentence covers four lines, while the others only cover three. Do you now understand my point?
> > 
> > 257476100-d285fecc-c666-475b-aecd-0d8639388e14
> 
> @Mayaleeeee the text beneath London is not the same length as the rest

Exactly my point, it's supposed to be the same length since the only difference there is the name of the location.

**Comment by thulieblack** [comment]
To solve this, is there a way we can align everything to the same width and height? That might solve the issue Maya is pointing out and make everything be in uniform

**Comment by AceTheCreator** [comment]
Pr updated! @thulieblack @Mayaleeeee @derberg

**Comment by thulieblack** [comment]
1. Madrid is now 19 October 
2. We need to improve mobile usability (at a later stage)

**Comment by AceTheCreator** [comment]
> 1. Madrid is now 19 October
> 2. We need to improve mobile usability (at a later stage)

Updated!

**Comment by Barbanio** [comment]
Hi, sorry for the delay.

@AceTheCreator, good job! It looks great!

1. When I use the menu, it remains on the screen, and I must close it manually. 

2. In the mobile version, in the Venues section, the text and images are cut off on the right side. 

IMG

**Comment by AceTheCreator** [comment]
> In the mobile version, in the Venues section, the text and images are cut off on the right side.

@Barbanio at what screen resolution is this?

**Comment by Barbanio** [comment]
1334x750px (326 ppp)

**Comment by AceTheCreator** [comment]
> 1334x750px (326 ppp)

Is it the last preview on the list?

**Comment by Barbanio** [comment]
> Is it the last preview on the list?

Yes

**Comment by AceTheCreator** [comment]
> > Is it the last preview on the list?
> 
> Yes

Oh! this was because I didn't include an infinity scroll. Fixed!

**Comment by Mayaleeeee** [comment]
1. The Madrid button should be active once the users reach the speaker section since that's the first set of speakers they will see.

Screenshot 2023-08-07 160238

2. The subscribe button here takes me to another page: **"Subscribe to our newsletter to receive news about AsyncAPI."** Is that intentional?

Screenshot 2023-08-07 160401

**Comment by Mayaleeeee** [comment]
> > Under this same page, we also mentioned that we should add a section where users will see the event/host sponsors of the location immediately after they enter the page, it's on the design I created (see the picture below), but it's not on the deploy-preview
> 
> @Mayaleeeee It doesn't make sense to me why we are gonna have the sponsors there and then still have it in another section of the page. That was why I didn't include it.
> 
> The remaining you pointed out(apart from the last one) was all intentional as I also want other contributors to fix this minor issue as a means of contributing to the project

We can reduce the whitespace since we won't add the event/host sponsors here.
Screenshot 2023-08-07 162359

File: config/city-lists.json [hunk 21]
```diff
@@ -0,0 +1,36 @@
...
+		"date": "October 19, 2023",
+		"description": "Join us in Madrid for AsyncAPI Conference and learn how to speak fluent API! Let's taco 'bout messaging and have a fiesta you won't forget!",
+		"img": "https://images.unsplash.com/photo-1539037116277-4db20889f2d4?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=1470&q=80",
+		"address": "Sngular Madrid, C. de Labastida, 1, 28034 Madrid, Spain",
+		"sponsors": ["/img/sngular.png"]
+	},
+	{
+		"name": "Bangalore",
+		"country": "India",
+		"date": "TBA",
```
Comment (thulieblack) [inline config/city-lists.json:23]:
```suggestion
		"date": "November 30, 2023",
```

File: config/city-lists.json [hunk 22]
```diff
@@ -0,0 +1,36 @@
...
+		"description": "Join us in Bangalore for the AsyncAPI Conference and learn more about APIs and EDAs! Let's have a memorable fiesta!",
+		"img": "https://images.unsplash.com/photo-1596176530529-78163a4f7af2?ixlib=rb-4.0.3&ixid=M3wxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8fA%3D%3D&auto=format&fit=crop&w=2454&q=80",
+		"address": "309, Venkatesh complex, floor, 100 feet road, Indiranagar, Bangalore, Karnataka 560038, India"
```
Comment (thulieblack) [inline config/city-lists.json:26]:
```suggestion
		"address": "310, 6th Main Rd, HAL 2nd Stage, Indiranagar, Bengaluru, Karnataka 560038, India"
```

**Comment by Mayaleeeee** [comment]
For each of the locations, we could have something like this:
 
 1. Madrid:
   Join us in **Madrid** for AsyncAPI Conference and learn how to speak fluent API! Let's taco about messaging and have a fiesta you will remember!

2. London:
   Join us in **London** for the AsyncAPI Conference and unlock the API of opportunities! Let's bridge the messaging gap and create a tech buzz you will want to experience!

3. Paris:
   Join us in **Paris** for the AsyncAPI Conference and experience the art of API perfection! Let's craft a symphony of messaging and celebrate tech in style!"

4. Bengaluru:
   Join us in **Bengaluru** for the AsyncAPI Conference and decode the language of APIs! Let's crack the language of APIs and ignite a tech revolution with some spice you'll remember!

The names of the locations will be displayed **in bold and in the same colour as the date.** This signifies that they are clickable. Users can click on the location name or the accompanying image to access the corresponding page directly. I recommend this approach to avoid confusion caused by the phrase **'Join us,'** which begins each location's description.  

I'd like more suggestions and would appreciate your input on this approach. @AceTheCreator @thulieblack @derberg @Barbanio

**Comment by AceTheCreator** [comment]
> > > Under this same page, we also mentioned that we should add a section where users will see the event/host sponsors of the location immediately after they enter the page, it's on the design I created (see the picture below), but it's not on the deploy-preview
> > 
> > 
> > @Mayaleeeee It doesn't make sense to me why we are gonna have the sponsors there and then still have it in another section of the page. That was why I didn't include it.
> > The remaining you pointed out(apart from the last one) was all intentional as I also want other contributors to fix this minor issue as a means of contributing to the project
> 
> We can reduce the whitespace since we won't add the event/host sponsors here. Screenshot 2023-08-07 162359
> 
done!
> I realized the map on the location page is on the preview website. Was that intentional? Screenshot 2023-08-07 163039

This is also intentional. It will come as a contribution.

**Comment by Mayaleeeee** [comment]
> > 1. The Madrid button should be active once the users reach the speaker section since that's the first set of speakers they will see.
> > 
> 
> This is intentional, as suggested by @derberg we should render all speakers by default and then by clicking you filter by region
> > Screenshot 2023-08-07 160238
> > 
> > 2. The subscribe button here takes me to another page: **"Subscribe to our newsletter to receive news about AsyncAPI."** Is that intentional?
> > 
> 
> Yea, this is also intentional as we don't have the bandwidth to maintain a different newsletter for conference
> 
> > Screenshot 2023-08-07 160401
> 
> 

We should have a "All" button then.
What do you think?

**Comment by thulieblack** [comment]
Hey @Mayaleeeee, the newsletter details are fine as it is. The monthly newsletter shipped out every month will contain all the necessary information about all the conferences

**Comment by Mayaleeeee** [comment]
> Hey @Mayaleeeee, the newsletter details are fine as it is. The monthly newsletter shipped out every month will contain all the necessary information about all the conferences

Ohh, that's cool then.

**Comment by AceTheCreator** [comment]
I think this PR is rtm cc @derberg @thulieblack @Mayaleeeee @Barbanio

**Comment by Mayaleeeee** [comment]
> I think this PR is rtm cc @derberg @thulieblack @Mayaleeeee @Barbanio

I apologize, @AceTheCreator, but there seems to be a misunderstanding. I was actually referring to the specific locations marked in the screenshot below and not the **main location header.**

Screenshot 2023-08-08 135904

**Comment by Mayaleeeee** [comment]
> lgtm, I personally do not spend much time on websites to be patient enough to wait for entire globe animation, and see how new cities and countdown shows up. I personally would prefer to show all locations and countdowns by default (and they can highlight or whatever one after another if you want). But if I'm the only one and others like how it is now, I won't block.
> 
> Great stuff @AceTheCreator 🚀 @Mayaleeeee you've got a pretty nice production implementation of your design for your portfolio, well done 👏🏼

Thank you so much.
Well done @AceTheCreator

**Comment by AceTheCreator** [comment]
> > I think this PR is rtm cc @derberg @thulieblack @Mayaleeeee @Barbanio
> 
> I apologize, @AceTheCreator, but there seems to be a misunderstanding. I was actually referring to the specific locations marked in the screenshot below and not the **main location header.**
> 
> Screenshot 2023-08-08 135904

Oh! If that's what you meant, then I don't think that will be necessary because making the whole card clickable is the best UX IMHO not the text within a sentence. I'll change the title back to white :)

**Comment by Mayaleeeee** [comment]
> > > I think this PR is rtm cc @derberg @thulieblack @Mayaleeeee @Barbanio
> > 
> > 
> > I apologize, @AceTheCreator, but there seems to be a misunderstanding. I was actually referring to the specific locations marked in the screenshot below and not the **main location header.**
> > Screenshot 2023-08-08 135904
> 
> Oh! If that's what you meant, then I don't think that will be necessary because making the whole card clickable is the best UX IMHO not the text within a sentence. I'll change the title back to white :)

Sure, that works as well. My worry was about the main location header, and I can see that you have adjusted the color. Thank you.

**Review by thulieblack — APPROVED** [review]
looks good; let's get the 1st iteration merged!!

**Comment by derberg** [comment]
merge merge merge and in a follow up PR I guess it is a right time to add CODEOWNERS

**Comment by Mayaleeeee** [comment]
:partying_face: :partying_face: :partying_face:

**Comment by Mayaleeeee** [comment]
🥳🥳🥳🌸

**Pull request merged**